src/filter-images.js

```diff
@@ -0,0 +1,21 @@
+function filterImages(images, filter) {
+    const text = filter.text.toLowerCase();
+    return images.filter(image => {
+
+        const title = image.title.toLowerCase();
+        const keyword = image.keyword.toLowerCase();
```

Comment:
When ```onAdd``` fires in your ```AddImage``` component, it returns an object like 
```const newImage = {
                url: formData.get('url'),
                title: formData.get('title'),
                description: formData.get('description')
            };
           onAdd(newImage);
```

Once added your ```filterImages``` function fails because you aren't returning a keyword property in your object to add. I would consider either getting rid of this line and not filter or changing your ```onAdd``` function to account for this edge case.